test/db.js

```diff
@@ -366,6 +367,13 @@ test('station', co(function* (t) {
 	t.end()
 }))
 
+test('line with additionalName', co(function* (t) {
+	const departures = yield(client.departures(potsdamHbf, { duration: 12*60, products: { bus: false, suburban: false, tram: false } }))
```

Comment:
Please remove the parenthesis after `yield` to make clear that `yield` is ECMAScript syntax, not a function.

Reply:
Fixed.